pandas/core/generic.py

```diff
@@ -9904,22 +9913,42 @@ def _check_percentile(self, q):
         2   -0.055556
         dtype: float64
 
-        See the percentage change in a Series where filling NAs with last
-        valid observation forward to next valid.
+        See how the computing of percentage change is performed in a Series
+        with NAs. With default `skipna=True`, NAs are dropped before the
+        computation and eventually the results are reindexed like the original
+        object, thus keeping the original NAs.
 
-        >>> s = pd.Series([90, 91, None, 85])
+        >>> s = pd.Series([90, 91, None, 85, None, 95])
```

Comment:
I realize this was in place before you but let's change `None` to `np.nan` to shy away from inference in the example

Reply:
can you update this